Mathlib/Order/WithBot.lean

```diff
@@ -806,11 +813,19 @@ alias coe_untop'_le := coe_untopD_le
 @[simp]
 theorem coe_top_lt [OrderTop α] : (⊤ : α) < x ↔ x = ⊤ := by cases x <;> simp
 
-lemma forall_gt_iff_eq_top : (∀ a : α, a < y) ↔ y = ⊤ := by
```

Comment:
Hold on, why are these getting removed?

Reply:
Well, in https://github.com/leanprover-community/mathlib4/pull/23101#discussion_r2004046934 you said you wanted  `y = ⊤ ↔ ∀ a : α, a < y` to exist. Since `(∀ a : α, a < y) ↔ y = ⊤` already existed then, I assumed you wanted me to replace it, so I have replaced it (and indeed I don't think we should have both).

Reply:
ehhh, okay